### ISSUE TITLE
Uncaught SyntaxError: Unexpected string

### ISSUE THREAD

**Issue by scozv** [mikaspell/gitbook-plugin-yametrika#1]
Hi @mikaspell,

when load your plugin, browser's console throws out an exception:

``` JavaScript
Uncaught SyntaxError: Unexpected string      plugin.js:7
```

I read your code in `plugin.js`

``` JavaScript
var yaCounter"+config.number+" = new Ya.Metrika({id: "+config.number+"});
```

It seems that there is a SyntaxError just after `var`.

**Comment by mikaspell** [comment]
Hi, thanks for issue

**Comment by mikaspell** [comment]
fixed in 1.6.6
